### PR TITLE
[#1799] Implement CMS plugin for mijn aanvragen

### PR DESCRIPTION
**Pull request by pi-sigma**
Taiga [#1799](https://taiga.maykinmedia.nl/project/open-inwoner/task/1799)

### CODE REVIEW

**Comment by alextreme** [comment]
Rebase on PR 804, after that discuss this with @Bartvaderkin

**Comment by pi-sigma** [comment]
Side note: I was tired of copy-pasting the boilerplate code required for tests that involve zaken, so I created a re-usable mixin. If we want to keep this we should modify other tests involving zaken as well. Otherwise the mixin should be removed.

File: src/open_inwoner/cms/cases/tests/test_plugin_cases.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        # anonymous user
+        html, context = cms_tools.render_plugin(CasesPlugin)
+
+        self.assertIsNone(context["cases"])
+
+        # check that html empty
+        with self.assertRaises(lxml.etree.ParserError) as ctx:
+            pq(html)
+
+        self.assertEqual(str(ctx.exception), "Document is empty")
```
Comment (Bartvaderkin) [inline src/open_inwoner/cms/cases/tests/test_plugin_cases.py:40]: @pi-sigma We don't really need lxml and pyquery to check if a string is empty :smile:

**Comment by pi-sigma** [comment]
Closing for now because the PR has been on hold for quite a while. Can be re-opened or scavenged if the issue is taken up again.

**Pull request closed without merging**